mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -5601,6 +5622,22 @@ LogicalResult ConstantMaskOp::verify() {
   return success();
 }
 
+bool ConstantMaskOp::isFullMask() {
```

Comment:
I would call this `isAllOnesMask` or similar to align with terminology used  in other places.